ADAL/src/ADAuthenticationError.m

```diff
@@ -87,7 +87,8 @@ - (id)initInternalWithDomain:(NSString *)domain
     {
         // If we're getting nil back here we have bigger problems and the logging below is going to fail anyways.`
         return nil;
-    }
+    }  
+    
     
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)